### PR TITLE
Adding missing dependency

### PR DESCRIPTION
**Pull request by cbargren**
Fixing barg support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
     "type": "git",
     "url": "git+https://github.com/maxkorp/behnam.git"
   },
+  "dependency": {
+    "barg": "1.0.0"
```
Comment (Mr-Wallet) [inline package.json:11]: Barg doesn't have a stable 1.0.0 release yet :-1:

Comment (chilblane) [reply package.json:11]: In fact, based on prior experience, I'd say `barg` is quite unstable.

**Comment by maxkorp** [comment]
It's not stable, but this project is "Agile"

**Pull request merged**